package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "bedrock-profile",
+  "version": "1.1.4-0",
+  "description": "Profiles for Bedrock Applications",
+  "main": "./lib",
+  "scripts": {},
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/bedrock-profile"
+  },
+  "keywords": [
+    "bedrock"
+  ],
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com/"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-profile/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-profile",
+  "dependencies": {
+    "assert-plus": "^1.0.0",
+    "axios": "^0.19.2",
+    "base64url-universal": "^1.0.0",
+    "bedrock-https-agent": "^1.0.1",
+    "bs58": "^4.0.1",
+    "cors": "^2.8.4",
+    "crypto-ld": "^3.5.2",
+    "did-method-key": "^0.4.0",
```

```suggestion
    "did-method-key": "^0.5.0",
```

This is a breaking change, not sure what impacts will be here https://github.com/digitalbazaar/did-method-key-js/blob/master/CHANGELOG.md